ios/keyman/Keyman/Keyman/AppDelegate.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  AppDelegate.swift
+//  Keyman
+//
+//  Created by Gabriel Wong on 2017-09-07.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
```

Comment:
This line replaces the old `main.m`; the main() function is thus auto-managed by Swift.  It likes doing this in the `AppDelegate` class.